src/test/java/uk/gov/companieshouse/ordernotification/orders/service/OrdersApiServiceImplTest.java

```diff
@@ -0,0 +1,127 @@
+package uk.gov.companieshouse.ordernotification.orders.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.function.Executable;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.handler.order.PrivateOrderResourceHandler;
+import uk.gov.companieshouse.api.handler.order.request.OrdersGet;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.api.model.order.OrdersApi;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.ordernotification.logging.LoggingUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+public class OrdersApiServiceImplTest {
```

Comment:
Rename test class OrdersApiOrderRetrieverTest

Reply:
Resolved in latest commit